### DISCUSSION TITLE
Database Schema Design and Modeling Tool (like DB Designer)

### DISCUSSION THREAD

**Discussion by stefroussos** [supabase/supabase#3036] [Feature Requests]
The Supabase Table Editor is a very interactive and visual Postgres interface, so a schema design tool like [DB Designer](https://www.dbdesigner.net/) could be a great addition, allowing users to visualise their databases in a UML type diagram.

**Comment by kangmingtay** [comment]
Hey @stefroussos, someone from the community was kind enough to build a db schema visualiser for us! You can check it out here (https://www.madewithsupabase.com/p/supabase-schema)

**Reply by stefroussos** [reply]
Hi @kangmingtay thanks! I'll check it out!
